src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/__tests__/autocomplete.command.stats.test.ts

```diff
@@ -405,9 +488,7 @@ describe('autocomplete.suggest', () => {
           });
         });
 
-        test('suggests `??function` option', async () => {
-          const { suggest } = await setup();
-
+        test('suggests `?function` option', async () => {
```

Comment:
Why did you change the test here? It should suggest ??function

Reply:
I didn't... maybe a merge thing. Will clean it up

Reply:
https://github.com/elastic/kibana/pull/216379/commits/866a5e41e331836150f1f575b1aaa279b2f0764f